syntax = "proto3";

message OrderMessage {
    int32 id = 1;
    int32 user_id = 2;            // ID of the user placing the order
    int32 product_id = 3;         // ID of the product being ordered
    int32 quantity = 4;           // Quantity of the product ordered
    float total_price = 5;        // Total price of the order
    string status = 6;            // Status of the order, e.g., "pending", "completed"
    string action = 7;            // Action, e.g., "created", "updated", "deleted"
}

